HISMalawi/BHT-EMR-API
# Location Metadata FIX

## Description

This pull request reinstates the Mangochi Medi Clinic location which was overwritten during recent updates to the metadata. The only record that was affected in the locations table.

## How to verify this issue

To verify this issue, follow these steps:

1. Start with the current tag v4.19.0 and update the metadata:
2. Open your MySQL workbench or another tool to access MySQL data.
3. Select the location with ID 1106:

```sql
SELECT * FROM location WHERE id = 1106;
```

4. Verify that the location name is `Kwidu`.
5. Switch to the previous release v4.18.0 and repeat steps 2-4.
6. Verify that the location name is `Mangochi Medi Clinic`.
7. Now switch to this branch and do the same
8. Verify that the location name is `Mangochi Medi Clinic`.

If the location name is `Mangochi Medi Clinic` in v4.18.0 and this branch,  and `Kwidu` in v4.19.0, then this pull request has successfully resolved the issue.

## Further Steps
You can choose to verify all other locations exists as well
